nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/impl/DocumentModelImpl.java

```diff
@@ -1515,4 +1515,16 @@ protected static SchemaManager getSchemaManager() {
         return schemaManager;
     }
 
+    @Override
+    public void move(DocumentRef dst, String name) {
+        DocumentModel newDoc = getSession().move(getRef(), dst, name);
```

Comment:
why create a variable here ?

Reply:
Just to decouple call to `move` from `refresh`.